### ISSUE TITLE
Expand ""Usage" paragraph in Readme

### ISSUE THREAD

**Issue by jwrossee** [technovangelist/airenamer#2]
HI, love this tool but not being a dev by trade, I couldn't get it to work for me locally, even after trying a few hours. Installed deno via homebrew and read the docs online and followed the installation part, but no success. Even asked codellama running locally with ollama. Please kindly provide a few more lines in the "usage" paragraph for people not familiar with running a deno project. I would love to get it working for me locally with ollama. Thanks in advance and also for sharing this! Highly appreciated.

**Comment by technovangelist** [comment]
so installing the app from releases didn't work? no need to install deno. what platform are you runing on

**Comment by jwrossee** [comment]
Apple M1 OS X 13.6.1. The error in the terminal is:
Command: ~/Dev/airenamer/airenamer
Return: error: Uncaught (in promise) SyntaxError: "undefined" is not valid JSON
    at JSON.parse (<anonymous>)
    at getkeywords (file:///home/runner/work/airenamer/airenamer/main.ts:20:25)
    at eventLoopTick (ext:core/01_core.js:183:11)
    at async file:///home/runner/work/airenamer/airenamer/main.ts:33:24
    
    P.S: The releases link in the Readme.md takes you to the OBM releases.

**Comment by jwrossee** [comment]
Ok, so I have been trying to troubleshoot all morning and after hours of error messages and learning about the deno framework along the way, I changed
`    "model": "llava:13b",`
to 
`    "model": "llava",`
and after compiling the executable is now working! Llava:13b wasn't offered as a download when I installed it, so I installed just Llava. I still get a different error message about file name too long, but that's another issue. As a side effect Deno has really grown on me, might do more with it :-)
